DateTimeOnly/packages.lock.json

```diff
@@ -8,6 +8,15 @@
         "resolved": "0.2.12-alpha",
         "contentHash": "uu3yAa6hQZJoJZfBwXSvh+bN03KO06sWNZXwodxz7J1wva1g1+O4TnWJxX4dmsP6tLDTZtKy8IHiPLOOZIwt9w=="
       },
+      "Microsoft.NETFramework.ReferenceAssemblies": {
```

Comment:
This will add the unnecessary dependency for all targets. We should avoid such situations for low-level multi-targeting NuGet packages.

Reply:
Removed the reference.